api/pkg/apis/v1alpha1/utils/parser.go

```diff
@@ -781,7 +781,7 @@ func (n *FunctionNode) Eval(context utils.EvaluationContext) (interface{}, error
 			if err != nil {
 				return nil, err
 			}
-			return context.SecretProvider.Get(FormatAsString(obj), FormatAsString(field), context)
+			return context.SecretProvider.Get(context.Context, FormatAsString(obj), FormatAsString(field), context)
```

Comment:
where does this context.Context come from? We should get it from the evaluation context?

Reply:
yes it is from evaluation context.